docs/pages/cloud/introduction.mdx

```diff
@@ -1,16 +1,60 @@
 ---
-title: Cloud
-description: Teleport as a service
-h1: Teleport cloud
+title: Teleport Cloud
+description: Teleport Cloud is a fully managed deployment of Teleport.
 videoBanner: 1jhKOtBinm4
 ---
 
-We run Teleport Cloud as hosted, managed Teleport as a service.
-Connect your nodes, web applications, kubernetes clusters and databases.
-Sign up [here](https://goteleport.com/get-started/).
+Teleport Cloud is a fully managed deployment of the Teleport Auth Service and
+Proxy Service. 
 
-## Next Steps
+When you [sign up for a Teleport Cloud account](https://goteleport.com/signup/), you will receive a subdomain of
+`teleport.sh` that is dedicated to your tenant and points to the Teleport Proxy
+Service. 
 
-- Get started with cloud [in 5 minutes](./getting-started.mdx).
-- Join the [Teleport Discussons](https://github.com/gravitational/teleport/discussions) and ask a question.
-- Join the [Slack channel](https://goteleport.com/slack).
+The following management tasks are handled for you:
+
+- Auth and Proxy Service upgrades, security patches, scaling, monitoring,
+  maintenance, and network configuration
+- Auth Service backend management
+- DNS record management
+- Certificate authority rotation
```

Comment:
Is this true?

Reply:
I should have checked this more carefully—looks like `tctl auth rotate` is in the hands of the user. In this case, I was thinking of the TLS credentials that the Proxy Service uses to communicate with clients, though I'm not sure that's worth mentioning here.